src/components/Map.svelte

```diff
@@ -8,6 +8,7 @@
 
   export let map: mapboxgl.Map | null = null
   export let markers: MapMarker[] = []
+  // default values for map that are overwritten when Map is called
```

Comment:
"that are overwritten when Map is called"

not true unless you pass these values as props. i think we can remove this comment